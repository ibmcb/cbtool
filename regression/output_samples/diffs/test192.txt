|experiment_id                        |EXP-07-22-2013-04-49-25-PM-EDT     |REGRESSION_A                       
