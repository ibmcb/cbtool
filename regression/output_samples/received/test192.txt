 status: Attribute "collect_from_host" was set to "false". Skipping Host OS performance monitor daemon startup
The global object "time" on Cloud TESTCLOUD was modified:
|"sub-attribute" (key)                |old value                          |new value                          
|experiment_id                        |EXP-07-22-2013-04-49-25-PM-EDT     |REGRESSION_A                       
