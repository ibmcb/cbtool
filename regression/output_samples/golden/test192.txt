 status: Attribute "collect_from_host" was set to "false". Skipping Host OS performance monitor daemon startup
The global object "time" on Cloud TESTCLOUD was modified:
|"sub-attribute" (key)                |old value                          |new value                          
|experiment_id                        |EXP-07-01-2013-01-56-48-PM-EDT     |REGRESSION_A                       
